apps/wallet-dashboard/components/dialogs/unstake/views/UnstakeTimelockedObjectsView.tsx

```diff
@@ -34,22 +39,35 @@ interface UnstakeTimelockedObjectsViewProps {
     onBack?: () => void;
 }
 
+const REDUCTION_STEP_SIZE = 5;
+
 export function UnstakeTimelockedObjectsView({
     groupedTimelockedObjects,
     onClose,
     onBack,
     onSuccess,
 }: UnstakeTimelockedObjectsViewProps) {
+    const [reductionSize, setReductionSize] = useState(0);
+    const [isMaxTransactionSizeError, setIsMaxTransactionSizeError] = useState(false);
```

Comment:
in `useGetSupplyIncreaseVestingObjects` var `isMaxTransactionSizeError` is used as ref    ` const isMaxTransactionSizeError = useRef(false);`
can we do the same thing here?